lib/cln.js

```diff
@@ -16,6 +16,8 @@ export const createInvoice = async ({ socket, rune, cert, label, description, ms
     agent = protocol === 'https:'
       ? new HttpsProxyAgent({ ...proxyOptions, ...httpsAgentOptions, rejectUnauthorized: false })
       : new HttpProxyAgent(proxyOptions)
+  } else if (process.env.NODE_ENV === 'development' && !cert) {
+    protocol = 'http:'
```

Comment:
I couldn't get the cert chain to validate in polar even after generating my own.